README.md

```diff
@@ -2619,6 +2619,15 @@ useGLTF(url, '/draco-gltf')
 useGLTF.preload(url)
 ```
 
+> **Note** <br>If you are using the CDN loaded draco binaries, you can get a small speedup in loading time by prefetching them.
```

Comment:
About `**Note**` syntax, I would add a small subtitle and a newline to separate:

```md
> **Note** <br>Link prefetching
> 
> If you are using the CDN loaded draco binaries, you can get a small speedup in loading time by prefetching them.
```

please ignore if you don't care, it's really nothing